.travis.yml

```diff
@@ -16,8 +16,11 @@ branches:
   except:
   - /^v\d/
 
+before_script:
+  - chmod +x build.sh
```

Comment:
We don't need this is you push +x status to Git.

Reply:
Cool! I didn't know this.

Reply:
Updated